sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerQueryTestSuite.scala

```diff
@@ -119,6 +115,7 @@ class ThriftServerQueryTestSuite extends SQLQueryTestSuite with SharedThriftServ
 
       configSet.foreach { case (k, v) =>
         statement.execute(s"SET $k = $v")
+        SQLConf.get.setConfString(k, v)
```

Comment:
SQLConf.get in the test thread is not the one `statement.execute` affected. It will be used in this thread for normalizing the binary result

Reply:
do you mean generating the binary pretty string happens outside of the `statement`?

Reply:
yes，like timestamp